### PR TITLE
Refactor the styles

### PR DESCRIPTION
**Pull request by AleksanderSkubala**
Take a look at some small changes I made in this refactor ;D

### CODE REVIEW

File: src/components/form.js [hunk 1]
```diff
@@ -4,8 +4,9 @@ import { Formik } from "formik"
 import { Form, Datepicker, SubmitBtn, Input, Select } from "react-formik-ui"
 import * as Yup from "yup"
 import moment from "moment"
+import styled from "styled-components"
 
-import "./form.css"
+const StyledForm = styled.div``
```
Comment (AdamSiekierski) [inline src/components/form.js]: I think we don't need that component :)

File: src/components/form.js [hunk 2]
```diff
@@ -154,7 +155,7 @@ export default class AddForm extends Component {
 
   render() {
     return (
-      <div className="form">
+      <StyledForm>
```
Comment (AdamSiekierski) [inline src/components/form.js]: You may just remove it, I think. Or replace it with just a `div` :D

File: src/components/form.js (hunk 1 above)
Comment (AleksanderSkubala) [reply src/components/form.js]: Well we had empty file `form.css` so I thought we will be doing some styles for it, but ya we can delete it ;)

File: src/components/form.js (hunk 2 above)
Comment (AdamSiekierski) [reply src/components/form.js]: I don't think it affects the website's appearance at all

Comment (AdamSiekierski) [reply src/components/form.js]: But check that out :)

**Comment by AdamSiekierski** [comment]
Except the comment I've given on the `StyledForm`...

LGTM🔥

Check out what happened here, and I think we're ready to go :)

**Comment by AdamSiekierski** [comment]
Ok, looks fine :D

**Comment by AleksanderSkubala** [comment]
🔥

**Comment by AdamSiekierski** [comment]
If TripleA was Gatsby, I would give you some sick swag :D

**Pull request merged**